staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -88,24 +89,24 @@ var (
 	// Can we make these passable to NewGenericWebhook?
 	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
 	// TODO(audit): figure out a general way to let the client choose their preferred version
-	groupVersions = []schema.GroupVersion{auditv1beta1.SchemeGroupVersion}
-	registry      = registered.NewOrDie("")
+	registry = registered.NewOrDie("")
 )
 
 func init() {
-	registry.RegisterVersions(groupVersions)
-	if err := registry.EnableVersions(groupVersions...); err != nil {
-		panic(fmt.Sprintf("failed to enable version %v", groupVersions))
+	allGVs := []schema.GroupVersion{auditv1alpha1.SchemeGroupVersion, auditv1beta1.SchemeGroupVersion}
```

Comment:
Codes in another place requires this init() function.
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit_test.go#L46
That naive code is wrote by me :(

I will get ride of the dependency when address another comment:
https://github.com/kubernetes/kubernetes/pull/49115#discussion_r133416080